setup.py

```diff
@@ -1,4 +1,8 @@
-from setuptools import setup, find_packages
+from __future__ import annotations
```

Comment:
So this is necessary correct?

Reply:
:| I missed that one my apologies when I refactored the yaml and corrected __main__.py

Reply:
Corrected and also removed from 'revChatGPT.py' as I missed on that one as well